djangocms_versioning/admin.py

```diff
@@ -249,12 +249,44 @@ def _get_edit_link(self, obj, request, disabled=False):
             }
         )
 
+    def _get_revert_link(self, obj, request, disabled=False):
+        """Helper function to get the html link to the revert action
+        """
+        if obj.state in (UNPUBLISHED, ARCHIVED):
+            pks_for_grouper = obj.versionable.for_grouper(
+                obj.grouper).values_list('pk', flat=True)
+            drafts = Version.objects.filter(
+                object_id__in=pks_for_grouper, content_type=obj.content_type,
+                state=DRAFT)
+        else:
+            # Don't display  =the link if it's a draft or published
+            return ''
+
+        if drafts.exists():
+            disable_revert_link = True
+            revert_url = ''
+        else:
+            disable_revert_link = False
+            revert_url = reverse('admin:{app}_{model}_revert_redirect'.format(
+                app=obj._meta.app_label, model=self.model._meta.model_name,
+            ), args=(obj.pk,))
+
+        return render_to_string(
+            'djangocms_versioning/admin/revert_icon.html',
+            {
+                'revert_url': revert_url,
+                'disabled': disabled,
```

Comment:
What's the difference between disabled and disable_revert_link? Seems like it could be the same variable